### PR TITLE
Added speaker with sound effects

### PR DESCRIPTION
**Pull request by moclei**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -61,7 +62,7 @@ void setup()
 {
     Serial.begin(57600);
     Serial1.begin(9600);
-
+    // setupSpeaker();
```
Comment (jordynchuhaloff) [inline src/main.cpp]: should this be here?

File: src/main.cpp [hunk 2]
```diff
@@ -260,6 +264,7 @@ void undoLastTurn() {
     turnList[turnCount-1] = -1;
     cellsState[cellIndex] = 0;
     turnCount--;
+    undoMusic();
```
Comment (jordynchuhaloff) [inline src/main.cpp]: what if we call this `playUndoSound` to match the others?

File: src/main.cpp [hunk 3]
```diff
@@ -174,6 +177,7 @@ void handleCellTriggered(int cellIndex, bool isUndo) {
     if (isUndo) {
         setCellColor(cellIndex, COLOR_OFF);
     } else {
+        playSound();
```
Comment (jordynchuhaloff) [inline src/main.cpp]: maybe `playCellTriggeredSound`?

**Review by jordynchuhaloff — COMMENTED** [review]
had some naming ideas

File: src/main.cpp (hunk 1 above)
Comment (moclei) [reply src/main.cpp]: no

Comment (moclei) [reply src/main.cpp]: should I commit again without it?

File: src/main.cpp (hunk 2 above)
Comment (moclei) [reply src/main.cpp]: ok changed

File: src/main.cpp (hunk 3 above)
Comment (moclei) [reply src/main.cpp]: Ok changed

File: src/main.cpp (hunk 1 above)
Comment (moclei) [reply src/main.cpp]: ok changed and committed

**Review by jordynchuhaloff — APPROVED** [review]
lgtm

**Pull request merged**